### DISCUSSION TITLE
Supporting locales

### DISCUSSION THREAD

**Discussion by xhejtman** [cloudnative-pg/cloudnative-pg#1169] [Q&A]
Hello, 

are locales, e.g., cs_CZ.UTF-8 somehow supported in running postgres container? Or one has to build his own container with locales enabled?

**Comment by sxd** [comment]
Hi @xhejtman

Yes, you need to create your own container image with the locales you want for now, you can use our postgres images as base and then just add a `RUN apt update; apt install -y locales-all` and that will do the job

**Comment by xhejtman** [comment]
Is there plan for solution at your side?

**Comment by sxd** [comment]
@xhejtman 

We're not planning to release another image with all the locales installed, we're a bit short of hands

**Reply by kamikaze** [reply]
does it really take so much time to add a single apt command into Dockerfile ?
